src/Http/Headers/src/SameSiteMode.cs

```diff
@@ -1,13 +1,16 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.Net.Http.Headers
 {
     // RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
     public enum SameSiteMode
```

Comment:
If you're in the mood, it would also be nice to have a doc comment for the type itself. Including a link to the RFC would be useful!

Reply:
I can include all of those changes as well. Sorry for being new, but could you help me find those docs?

Reply:
Hi @natery2000 sorry I'm not sure I understand. I'm suggesting adding a new triple-slash (`///`) doc for the `public enum` section as well. So, much like the `///` docs you already added (thank you!), it's possible to add top-level docs for the type itself. And I'm suggesting deleting the current `//` comment, and making it a `///` comment with the same RFC link that's currently there. Hopefully that makes sense 😄

Reply:
Ah yes, that makes a lot of sense. I was thinking there was a formal external documentation that this needed to be included in. I will work on this tonight, thank you very much for the help.

Reply:
Thanks again!